conda/cli/install.py

```diff
@@ -201,7 +201,10 @@ def install(args, parser, command='install'):
     specs = []
     if args.file:
         for fpath in args.file:
-            specs.extend(common.specs_from_url(fpath, json=context.json))
+            try:
+                specs.extend(common.specs_from_url(fpath, json=context.json))
+            except:
+                raise CondaError("Error parsing file, likely due to invalid file format")
```

Comment:
This wording is fine, IMHO, but it would be good to add a link to what exactly constitutes a valid file (in the docs, ideally).  A lot of the confusion that people have is around when it is a YAML file they should be using vs. some text file.  Giving them pointers in the error message will be very helpful.